src/bootlint.js

```diff
@@ -828,14 +828,22 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.card-title` must have a `.card` ancestor.', badCardTitle);
         }
     });
-    /*
     addLinter('E027', function lintTableResponsive($, reporter) {
-        var badStructure = $('.table.table-responsive, table.table-responsive');
+
+        var tableSelectors = ['.table', 'table'];
+        var badStructureSelectors = [];
+
+        for (var i = 0; i < tableSelectors.length; ++i) {
```

Comment:
Any specific reason you went with prefix increments here and below? Also, I usually prefer to cache the len to a temp var.

Reply:
Nah, just my style (when in doubt, so prefix) which I got used to when I did crazy pointer arithmetic in C but I can change it for consistency. `.length` isn't cached anywhere else so I won't do it. I generally only cache it method calls (I know the lines are blurry sometimes).

Reply:
Please change to postfix then. I don't think we use prefix anywhere else.

As for length, true, I thought I made that change but apparently I didn't. So all good there.